Omise/Omise.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <PackageId>Omise</PackageId>
-    <PackageVersion>2.7.14</PackageVersion>
+    <PackageVersion>2.7.8</PackageVersion>

```

Comment:
Why are we bumping to lower version? Do you You mean `2.8.0`.
And I think it should do with separate PR right? Example [here](https://github.com/omise/omise-dotnet/commit/cf446d6454c6b7246bbe51a7afe18da293057cdc).

cc @danfowler

Reply:
Yeah I meant 2.8.0, should we really do a separate PR?

Reply:
Yes, we let's bump in a separate PR. Also, yes, the version is very wrong.

Reply:
ok

Reply:
Reversed the Bump, will do a separate PR for release once test cases are ready from @annatotest